geoips/commandline/test_interfaces.py

```diff
@@ -21,6 +21,9 @@
 FAILED_INTERFACE_HEADER_PRE = "FAILED: issues found within "
 FAILED_INTERFACE_HEADER_POST = "interface:"
 
+FAILED_INTERFACE_HEADER_PRE = "FAILED: issues found within "
```

Comment:
This was added back in after I removed it in v1.11.3-release...

Reply:
removed this